tsconfig.test.json

```diff
@@ -2,7 +2,7 @@
     "extends": "./tsconfig",
     "compilerOptions": {
         "declarationDir": null,
-        "declaration": false
-    },
-    "include": ["src/**/*.spec.ts"]
+        "declaration": false,
+        "allowJs": true
```

Comment:
I'm not really seeing any javascript files in src/ when I search in the repo, maybe I'm not searching correctly.

If we don't have any javascript files, why are we alloying JS?

Reply:
Ok this one is a trickster: It's because the unmocked dependencies are in JS ;)